src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventPipe.Internal.cs

```diff
@@ -9,6 +9,7 @@ namespace System.Diagnostics.Tracing
 {
     internal static partial class EventPipeInternal
     {
+#if FEATURE_EVENT_TRACE
```

Comment:
The eventpipe qcalls are guarded by FEATURE_PERFTRACING in runtime, so maybe the same define should be used here?